<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>710</width>
        <height>490</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>710</width>
        <height>490</height>
       </size>
      </property>
      <widget class="QLabel" name="label_plain">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Plain_Pin:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_cipher">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Cipher_Pin:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_plain">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>500</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_cipher">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>500</width>
         <height>400</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_encrypt">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>10</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Encrypt</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_decrypt">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>150</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Decrypt</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_clear">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>220</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_SHA256">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>80</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>SHA256(MD5)</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
